grapi/backend/kopano/message.py

```diff
@@ -119,6 +129,7 @@ class MessageResource(ItemResource):
         'from': lambda item, arg: set_user_email(item, 'from_', arg),
         'sender': lambda item, arg: set_user_email(item, 'sender', arg),
         'isRead': lambda item, value: update_attr_value(item, "read", value),
+        'categories': lambda item, value: set_categories(item, value),
```

Comment:
```suggestion
        'categories': lambda item, value: update_attr_value(item, "categories", value),
```
Please use `update_attr_value` instead.

Reply:
Should be fixed in https://github.com/Kopano-dev/grapi/pull/4/commits/ad62849ee1aa29192de26dc0751fa7959b05cccf